### PR TITLE
New snippets for the CTCMS SLURM cluster at NIST

### PR DESCRIPTION
**Pull request by reid-a**
Add snippets for the CTCMS SLURM cluster.

### CODE REVIEW

**Comment by reid-a** [comment]
The proximate cause of the build failure is that a snippet config file can't be found.  The path includes "SIGMA2_Norway_slurm", but in the repo attempting the merge, those snippets are apparently in "Norway_SIGMA2_SAGA_slurrm".

I can rename the directory in the PR branch, is that a fix? I'm uncertain if this is maybe a deeper build-system issue.

**Comment by tkphd** [comment]
Please wait for the fix in #187, then rebase your branch.
Also, the CTCMS snippet library does not get tested. Please edit `.travis.yml` and add a new section, to make sure everything would build correctly. Thanks!

**Comment by mikerenfro** [comment]
@reid-a #187 is now merged.

File: _config.yml [hunk 1]
```diff
@@ -20,23 +20,23 @@
 # installation. Remember to keep the leading slash on the
 # `snippets` variable below!
 
-snippets: "/snippets_library/ComputeCanada_Graham_slurm"
```
Comment (tkphd) [inline _config.yml:23]: Looks like the top-level `config.yml` has been overwritten. Please revert. Local config should go in the `_config_options.yml` file in the folder for your site's snippets.

**Comment by reid-a** [comment]
This PR now is rebased against the main repo including #188 and #187, and includes the CTCMS build in the CI. If the build passes, ti should be trivial to merge.

File: .travis.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
     - stage: "Build the site for SIGMA.2"
```
Comment (tkphd) [inline .travis.yml:46]: Please insert a "stage" for NIST CTCMS around here, so that the site gets test-built.

Comment (tkphd) [reply .travis.yml:46]: Resolved by 24092316c33b07b791b7bf48fcb45f858f7ac451

**Comment by tkphd** [comment]
@reid-a, thanks for diligently applying the changes to #187 as they happened -- the thought is appreciated.
For future PRs, please don't do this, as it inserts unnecessary duplicate commits when both PRs get merged into the gh-pages branch. Better practice is to wait for the other PR to get merged, then rebase your PR against the new gh-pages. This produces a cleaner commit history, and requires less human intervention.

File: _config.yml (hunk 1 above)
Comment (reid-a) [reply _config.yml]: Resolved by 76483ab.

**Comment by reid-a** [comment]
For the info of other users, I didn't actually manually track the other merge, I did the rebase workflow. There was an anomalous merge that went with that, possibly related to a gitlab config file that caused some earlier confusion.

I endorse the best-practice recommendation from @tkphd, with the additional remark that it apparently can still surprise you.

**Comment by tkphd** [comment]
Apologies for implicitly disparaging your Git skills, @reid-a!

**Pull request merged**